packages/grid/src/HeaderCell.tsx

```diff
@@ -30,25 +39,84 @@ export function HeaderCell<T>(props: HeaderCellProps<T>) {
   const { ref, isFocusableContent, onFocus } =
     useFocusableContent<HTMLTableHeaderCellElement>();
 
+  const { onClickSortColumn, setSortByColumnId, sortOrder, sortByColumnId } =
+    useColumnSortContext();
+
+  const valueAlignRight = align === "right";
+
+  interface HeaderCellSortingIconProps {
+    justify: FlexContentAlignment;
+  }
+
+  const HeaderCellSortingIcon = ({ justify }: HeaderCellSortingIconProps) => {
+    const className = withBaseName("sortingIcon");
+    const icon = (
+      <div
+        className={className}
+        style={{ display: "flex", justifyContent: `${justify}` }}
```

Comment:
We try to avoid inline styles. Can this be done via CSS?

Reply:
noted and yes!